services/movie.py

```diff
@@ -1,10 +1,12 @@
 from django.db.models import QuerySet
+from typing import Optional
+
 
 from db.models import Movie
```

Comment:
there is a specific order of different imports + import groups should be splited by 1 blank line